### PR TITLE
feat: migrated to shiki and fixed font-family

### LINKED ISSUE #6337: The event loop ascii art is broken
### URL:

https://nodejs.org/en/guides/event-loop-timers-and-nexttick/

### Browser Name:

Chrome

### Browser Version:

121.0.6167.160 (Official Build) (arm64)

### Operating System:

macOS

### How to reproduce the issue:

1. Visit https://nodejs.org/en/guides/event-loop-timers-and-nexttick/
2. Have the ascii art correctly as it used to be https://web.archive.org/web/20231226234524/https://nodejs.org/en/guides/event-loop-timers-and-nexttick/
3. ascii art is broken

The code is ok, but I think `shikiji` is trimming empty spaces

Screenshot 2024-02-11 at 9 03 49 PM

### PR DESCRIPTION
**Pull request by ovflowd**
## Description

This PR migrates us back to Shiki as Shikiji (the ESM-rewrite) got ported back to Shiki itself.
This PR also fixes that CodeBoxes without any specific language to highlight (default plain text) should not use th IBM-Plex font as it breaks plain text.

## Validation

Syntax Highlighting and Codeboxes should still work as usual

## Related Issues

Fixes https://github.com/nodejs/nodejs.org/issues/6337

### CODE REVIEW

**Comment by ovflowd** [comment]
cc @nodejs/nodejs-website

File: COLLABORATOR_GUIDE.md [hunk 1]
```diff
@@ -453,11 +453,9 @@ MDX is becoming the standard for parsing human-content on React/Next.js-based Ap
 - `rehype-autolink-headings`: Allows us to add Anchor Links to Markdown Headings
 - `rehype-slug`: Allows us to add IDs to Markdown Headings
 
-#### Syntax Highlighting (Shikiji) and Vercel
+#### Syntax Highlighting (Shiki) and Vercel
 
-We use [Shikiji][] which is a refactor of the famous [Shiki](https://github.com/shikijs/shiki) syntax highlighter in ESM. We use it to support our native ESM-nature, and since Shiki is incompatible on serverless environments and Edge functions due of the need of Node's `fs`. Shikiji is definitely a nice port/rewrite of Shiki which supports our needs.
-
-Shikiji is integrated on our workflow as a Reype Plugin, see the `next.mdx.shiki.mjs` file. We also use the `nord` theme for Shikiji and a subset of the supported languages as defined on the `shiki.config.mjs` file.
+Shiki is integrated on our workflow as a Reype Plugin, see the `next.mdx.shiki.mjs` file. We also use the `nord` theme for Shiki and a subset of the supported languages as defined on the `shiki.config.mjs` file.
```
Comment (mikeesto) [inline COLLABORATOR_GUIDE.md]:
```suggestion
Shiki is integrated on our workflow as a Rehype Plugin, see the `next.mdx.shiki.mjs` file. We also use the `nord` theme for Shiki and a subset of the supported languages as defined on the `shiki.config.mjs` file.
```

**Review by mikeesto — APPROVED** [review]
LGTM!

**Review by aymen94 — APPROVED** [review]

**Comment by ovflowd** [comment]
Merging as a fix for both the CodeBoxes and MDX rendering

**Pull request merged**